torch/functional.py

```diff
@@ -651,7 +648,7 @@ def _unique_impl(input: Tensor, sorted: bool = True,
             return_counts=return_counts, dim=dim)
 
     if dim is not None:
-        output, inverse_indices, counts = _VF.unique_dim(
+        output, inverse_indices, counts = _VF._unique_dim(
```

Comment:
@eellison do you see any potential problems with TorchScript for this change? I'm wondering if users actually see `at::unique_dim` serialized in a graph.

@jbschlosser since you've been through many BC issues in the past do you have a sense of what could break here as well?